### PR TITLE
ATL-546: document invoices.list date_before and date_after

### PR DESCRIPTION
**Pull request by joaoinacio**
JIRA: https://teamleader.atlassian.net/browse/ATL-546

### CODE REVIEW

**Review by BrechtBonte — DISMISSED** [review]

**Review by jnstr — DISMISSED** [review]

**Comment by MathiasDL** [comment]
I'm not 100% on board with this naming, see discussion here: https://teamleader.atlassian.net/browse/ATL-546

**Comment by joaoinacio** [comment]
Updated for consistency with the field name `invoice_date`

File: apiary.apib [hunk 1]
```diff
@@ -2699,6 +2700,8 @@ Get a list of invoices.
             + updated_since: `2016-02-04T16:44:33+00:00` (string, optional)
             + purchase_order_number: `000023` (string, optional)
             + payment_reference: `+++084/2613/66074+++` (string, optional)
+            + invoice_date_after: `2020-01-01` (string, optional)  - This date is inclusive
+            + invoice_date_before: `2021-12-31` (string, optional)
```
Comment (MathiasDL) [inline apiary.apib]: Maybe note here as well if it is inclusive or exclusive

**Review by BrechtBonte — DISMISSED** [review]

**Review by adognameddog — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by jnstr — DISMISSED** [review]

File: apiary.apib (hunk 1 above)
Comment (joaoinacio) [reply apiary.apib]: > Maybe note here as well if it is inclusive or exclusive

I've double checked and its actually also inclusive 🙈

**Review by jnstr — DISMISSED** [review]

**Review by adognameddog — DISMISSED** [review]

**Review by MathiasDL — DISMISSED** [review]

**Review by ArnaudWeyts — DISMISSED** [review]

**Review by BrechtBonte — DISMISSED** [review]

**Review by jnstr — APPROVED** [review]

**Review by BrechtBonte — APPROVED** [review]

**Review by ArnaudWeyts — APPROVED** [review]

**Pull request merged**